spec/requests/contacts_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe "Contacts", type: :request do
+  let(:user) { FactoryBot.create(:user) }
+  let(:contact) { FactoryBot.create(:contact, user:) }
```

Comment:
isnt this "user: user" or just "user" without the  :

Reply:
No, that's the whole thing of dominating the hash value, no?

Reply:
@xxjmsxx  Yeah, the `user:` implies the value is also a `user` variable defined before. In this case the line before.

Reply:
@emmvs \
~~dominating~~ omitting

Reply:
😂 typo